lib/ansible/modules/web_infrastructure/jenkins_script.py

```diff
@@ -142,24 +165,35 @@ def get_crumb(module):
 
 def main():
 
+    argument_spec = url_argument_spec()
+    argument_spec.update(
+        script=dict(required=True, type="str"),
+        url=dict(required=False, type="str", default="http://localhost:8080"),
+        validate_certs=dict(required=False, type="bool", default=True),
+        user=dict(required=False, no_log=True, type="str", default=None),
+        password=dict(required=False, no_log=True, type="str", default=None),
+        timeout=dict(required=False, type="int", default=10),
+        params=dict(type='dict'),
```

Comment:
This needs to be marked as not required.

Reply:
Got it!

Reply:
I think `False` is already the default as was used in `jenkins_plugin`:

https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/web_infrastructure/jenkins_plugin.py#L739

I will double check.

Reply:
Here is where the default comes from: https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py#L1512

```
            required = v.get('required', False)
```